pytradfri/__main__.py

```diff
@@ -17,10 +18,11 @@
 
 CONFIG_FILE = "tradfri_standalone_psk.conf"
 
-# pylint: disable=invalid-name, not-an-iterable, unsubscriptable-object
+# pylint: disable=not-an-iterable, unsubscriptable-object
 
 
-if __name__ == "__main__":
+def main() -> None:
```

Comment:
This script is meant to be run interactive so that variables are able to be accessed. This won't work if we hide them inside the functions.